docs/README.md

```diff
@@ -54,6 +54,34 @@ By default implementation we mean that whenever you use `checkout-summary` in yo
 
 Therefore, in order to customize the Checkout Summary configuration, you can simply use the default implementation in your code and change it as you wish.
 
+Additionally, you can also include a `"drawer"` block inside the `checkout-summary`, and the drawer trigger
+will appear alongside the "Summary" title.
+
+```jsonc
+{
+  "checkout-summary": {
+    "blocks": [
+      "drawer#my-drawer"
+    ],
+    "children": [
+      "flex-layout.row#summary-coupon",
+      "summary-totalizers"
+    ]
```

Comment:
I had no idea that it could accept both blocks and children. 😮

Reply:
I know, right?